pt/endpoints/address/model.py

```diff
@@ -73,14 +51,18 @@ def renew(time):
         index=int(time.strftime("%s")), count=AMI.query.count()
     )["addresses"]
     for amis in AMI.query.all():
-        amis.iota_address = str(new_address[amis.id])
+        amis.iota_address = str(new_address[amis.id - 1])
```

Comment:
Why do we need  '-1' ?  we can add a comment to it.

Reply:
Fix in 2d5b37f.